CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file.
 
-## [2.0.0] - TBD
+## [1.4.8] - 2021-01-29
```

Comment:
Is any of this breaking changes?

Reply:
Yes, View Details page depends on the singular workspace GET call (The one API that was updated). For this to work on existing member accounts, they would need re-onboarding

Reply:
Even disabling the built-in functionality is a breaking change but since it isn't being used by our customers and was disabled by default- it was deemed safe to remove. Are you suggesting a 2.0.0 release instead?

Reply:
Yes

Reply:
```suggestion
## [2.0.0] - 2021-01-29
```